charts/argo-rollouts/ci/enable-extra-objects.yaml

```diff
@@ -0,0 +1,14 @@
+# Test with extraObjects enabled
+# Do not deploy the CRDs as they are already present from the previous test
+installCRDs: false
+
+extraObjects:
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: datadog
+    type: Opaque
+    data:
```

Comment:
I think you need to use `stringData` here or the values of your dummy data needs to be base64 encoded

Reply:
@mkilchhofer you nailed it.  Thank you.

Reply:
Perfect. Now you need to rename the file. When CI should pick it up, it needs to end with `...-values.yaml` :)